onnxruntime/core/platform/posix/env.cc

```diff
@@ -359,6 +359,10 @@ class PosixEnv : public Env {
     return S_ISDIR(sb.st_mode);
   }
 
+  std::string GetRuntimePath() const override {
```

Comment:
I don't see this getting used anywhere.

Reply:
it is used when initialize the shared provider bridge.